src/main/scala/firrtl/backends/experimental/smt/StutteringClockTransform.scala

```diff
@@ -3,13 +3,32 @@
 
 package firrtl.backends.experimental.smt
 
-import firrtl.{ir, CircuitState, DependencyAPIMigration, Namespace, PrimOps, RenameMap, Transform, Utils}
-import firrtl.annotations.{Annotation, CircuitTarget, PresetAnnotation, ReferenceTarget, SingleTargetAnnotation}
+import firrtl.{
+  ir,
+  CircuitState,
+  DependencyAPIMigration,
+  Namespace,
+  PrimOps,
+  RegKind,
+  RenameMap,
+  SourceFlow,
+  Transform,
+  Utils
+}
+import firrtl.annotations.{
+  Annotation,
+  CircuitTarget,
+  PresetAnnotation,
+  PresetRegAnnotation,
+  ReferenceTarget,
+  SingleTargetAnnotation
+}
```

Comment:
I'm not sure what the limit should be, but these are approaching "should just use wildcard import"

Reply:
I used to be a lot more into "only import what you need", but yeah, I have come around and nowadays I would totally just use a wildcard import